hermes/bin/src/main.rs

```diff
@@ -2,17 +2,48 @@
 
 mod app;
 mod event;
+mod logger;
 mod reactor;
 mod runtime_context;
 mod runtime_extensions;
 mod wasm;
 
+use std::process;
+
+use tracing::{error, info, instrument};
 #[cfg(feature = "bench")]
 pub use wasm::module::bench::{
     module_hermes_component_bench, module_small_component_bench,
     module_small_component_full_pre_load_bench,
 };
 
+build_info::build_info!(fn build_info);
+
+// Disable process exit for clippy.
+#[allow(clippy::exit)]
+#[instrument]
 fn main() {
-    println!("Hello, world!");
+    // Initialize logger.
+    if let Err(err) = logger::init(logger::LogLevel::Info, true, true, true) {
+        println!("Error initializing logger: {err}");
+    }
+    // Create a new reactor instance.
+    let reactor_result = reactor::HermesReactor::new(Vec::new());
+    let mut reactor = match reactor_result {
+        Ok(reactor) => reactor,
+        Err(err) => {
+            error!("Error creating reactor: {}", err);
+            process::exit(1);
+        },
+    };
+
+    // Get build info string.
+    let build_info_str = format!("{:?}", build_info());
```

Comment:
Put this after initialising the logger and before doing anything else, like setting up the hermes reactor.